tests/Feature/CourseObserverTest.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Course;
+use App\CourseGrade;
+use App\CourseStatus;
+use App\Instructor;
+use App\Nimbus\Institution;
+use App\Registration;
+use App\SchoolTerm;
+use App\Subject;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use DatabaseSeeder;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Tests\Feature\Helpers\Auth\SetupUser;
+use Tests\TestCase;
+
+class CourseObserverTest extends TestCase
+{
+  use RefreshDatabase, WithFaker, SetupUser, WithoutMiddleware;
```

Comment:
@oshkbello remember when 403 were being returned from the api endpoints its not in the decent docs but apparently `WithoutMiddleware` can be used to disable middleware in tests